sdks/python/run_pylint.sh

```diff
@@ -29,6 +29,12 @@ BASE_BRANCH=python-sdk
 set -e
 set -o pipefail
 
+# This script runs pylint on changed files only and uses git for that purpose.
+# Skip this step if git is not available or this is not a valid git repository.
+(command -v git && git remote) || {
+  echo "Not running pylint. Not in a git repository."; exit 0; 
```

Comment:
I removed the hardcoded remote name. I think brings down to one failure mode, that is this is a shell script and would not run in Windows. As an option we can disable running pylint from maven.

That said, I think this change is useful because the script would not fail any more in tarballs.